src/config/schema.ts

```diff
@@ -98,6 +101,7 @@ export const configSchema = z
   .object({
     sponsorWalletMnemonic: z.string().refine((mnemonic) => ethers.utils.isValidMnemonic(mnemonic), 'Invalid mnemonic'),
     chains: chainsSchema,
+    fetchInterval: z.number().positive().optional().default(10),
```

Comment:
I think we want to be explicit with all our configuration values, so I'd remove the default value.